sql/catalyst/src/main/java/org/apache/spark/sql/catalog/v2/Catalogs.java

```diff
@@ -50,8 +51,10 @@ private Catalogs() {
    */
   public static CatalogPlugin load(String name, SQLConf conf)
       throws CatalogNotFoundException, SparkException {
-    String pluginClassName = conf.getConfString("spark.sql.catalog." + name, null);
-    if (pluginClassName == null) {
+    String pluginClassName;
```

Comment:
This is a temporary fix for loading session catalog. It might be fixed when there is a conclusion for the conf spark.sql.catalog.session .